src/windows/top_toolbar.c

```diff
@@ -1461,10 +1462,11 @@ static void window_top_toolbar_scenery_tool_down(short x, short y, rct_window *w
 	case SCENERY_TYPE_SMALL:
 	{
 		int quantity = 1;
-		if (window_scenery_is_build_cluster_tool_on) {
-			quantity = 35;
+		bool isCluster = window_scenery_is_build_cluster_tool_on && network_can_perform_action(network_get_current_player_group_index(), GAME_COMMAND_TOGGLE_SCENERY_CLUSTER);
+		if (isCluster) {
+			quantity = 36;
```

Comment:
@wolfreak99 we would like to merge this in for 0.0.4 due to the number of complains about abusers of this. Have you tried implementing it like @zsilencer suggested?